decidim-collaborative_texts/lib/decidim/collaborative_texts/test/factories.rb

```diff
@@ -15,4 +15,16 @@
     manifest_name { :collaborative_texts }
     participatory_space { create(:participatory_process, :with_steps, skip_injection:, organization:) }
   end
+
+  factory :collaborative_text_document, class: "Decidim::CollaborativeTexts::Document" do
+    transient do
+      skip_injection { false }
+    end
+    title { "Default title" }
```

Comment:
You could use this to make sure there are no xss injections ... but u may need i18n version `generate_localized_title`
```suggestion
    title { generate_title(:collaborative_text_document_title, skip_injection:)}
```